x-pack/plugins/actions/server/builtin_action_types/xmatters.ts

```diff
@@ -35,26 +35,11 @@ export type ActionTypeConfigType = TypeOf<typeof ConfigSchema>;
 // secrets definition
 export type ActionTypeSecretsType = TypeOf<typeof SecretsSchema>;
 const secretSchemaProps = {
-  user: schema.nullable(schema.string()),
-  password: schema.nullable(schema.string()),
+  user: schema.maybe(schema.string()),
+  password: schema.maybe(schema.string()),
```

Comment:
Using `schema.maybe` instead of `schema.nullable` here. Is that ok @pmueller?

Reply:
No, will need to be using `schema.nullable()` here.  `schema.maybe()` allows `undefined` values to be passed through, but `schema.nullable()` will convert them to `null`.  We need the `null` value to allow "erasing" an existing persisted value - using `undefined` leaves the persisted value the same.

Reply:
Cool thanks for explaining. I thought it was something like that but couldn't articulate it :)